python/ql/test/library-tests/filters/tests/Filter.expected

```diff
@@ -1,6 +1,13 @@
-| test.py:4:1:4:23 | Class MyTest |
-| test.py:6:5:6:21 | Function test_1 |
-| test.py:9:5:9:21 | Function test_2 |
```

Comment:
Is the fact that these are no longer matched intended? If so, then maybe we should just get rid of `test.py` entirely?

Reply:
yeah I noted in https://github.com/github/codeql/pull/9368/commits/a8b4b6a374f2528400111f5e7529957d5c068fd1: 

> Notice that although we loose the contrived examples in `test.py`, we do
gain support for real-world test-case construction, which seems worth
the tradeoff.

I think I'll delete that file :+1:

Reply:
Ah! I didn't see that commit message. Thanks!